x-pack/legacy/plugins/lens/public/indexpattern_plugin/operations/definitions/date_histogram.tsx

```diff
@@ -7,31 +7,26 @@
 import React from 'react';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
-import { EuiForm, EuiFormRow, EuiRange, EuiSwitch } from '@elastic/eui';
+import { isValidInterval } from 'ui/agg_types/utils';
```

Comment:
Imports from `ui/<something>` won't be possible in the new platform, maybe this needs to be passed in from the plugin?